### PR TITLE
Fix version mismatch in 1.11 branch

### PR DESCRIPTION
**Pull request by AkshaySainiDell**
#### Is this a new chart?

Yes/No

#### Which issue(s) is this PR associated with:

- https://github.com/dell/csm/issues/1221

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by suryagupta4 — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

File: charts/cosi/Chart.yaml [hunk 1]
```diff
@@ -27,10 +27,10 @@ type: application
...
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.3.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: 1.2.0-alpha
+appVersion: 0.3.0-alpha
```
Comment (shanmydell) [inline charts/cosi/Chart.yaml:36]: @AkshaySainiDell : Please re-verify this change

File: charts/csm-application-mobility/Chart.yaml [hunk 2]
```diff
@@ -6,13 +6,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
+version: 1.1.0
```
Comment (shanmydell) [inline charts/csm-application-mobility/Chart.yaml:9]: Please revert this as discussed

File: charts/csm-application-mobility/values.yaml [hunk 3]
```diff
@@ -70,7 +70,7 @@ velero:
 
   initContainers:
     - name: dell-custom-velero-plugin
-      image: dellemc/csm-application-mobility-velero-plugin:v0.3.0
```
Comment (shanmydell) [inline charts/csm-application-mobility/values.yaml:73]: please revert this

File: charts/csm-application-mobility/Chart.yaml (hunk 2 above)
Comment (AkshaySainiDell) [reply charts/csm-application-mobility/Chart.yaml]: Done

File: charts/csm-application-mobility/values.yaml (hunk 3 above)
Comment (AkshaySainiDell) [reply charts/csm-application-mobility/values.yaml]: done

File: charts/cosi/Chart.yaml (hunk 1 above)
Comment (AkshaySainiDell) [reply charts/cosi/Chart.yaml:36]: This is correct. I have taken reference from pervious release : [cosi-0.1.2](https://github.com/dell/helm-charts/releases/tag/cosi-0.1.2) https://github.com/dell/helm-charts/blob/d6ed125b36e648e6b0a9a37f49c86cd759a9f914/charts/cosi/Chart.yaml#L36

**Review by shanmydell — APPROVED** [review]

**Review by suryagupta4 — APPROVED** [review]

**Pull request merged**